assets/sass/04-elements/forms.scss

```diff
@@ -50,6 +50,10 @@ input[type="color"] {
 	height: calc(4 * var(--form--spacing-unit));
 }
 
+input:placeholder-shown {
```

Comment:
In Firefox this was affecting inputs with empty placeholders too — this should work in a more cross-browser friendly way:
```suggestion
input[placeholder]:not([placeholder=""]) {
```